goerli-shadow-fork/inventory/inventory.ini

```diff
@@ -0,0 +1,44 @@
+goerli-shadow-fork-lighthouse-geth-3	ansible_host=104.248.134.211
+goerli-shadow-fork-lighthouse-geth-2	ansible_host=104.248.141.102
+goerli-shadow-fork-lighthouse-geth-1	ansible_host=206.189.55.55
+
```

Comment:
Hey! hmm interesting..a shadow fork test :thinking: Thinking I have somewhat of an idea as to what this entails but definitely curious to understand further.

Also ok/possible for community members to contribute resources to efforts like this? (of course by sharing access keys and ensuring accessibility)

Reply:
Yess, so you can find some more info on the topic on twitter: https://twitter.com/parithosh_j/status/1484258074399592450

The idea is to spin up a testnet that follows goerli, but the testnet has a modified genesis config => So it forks off and merges with the CL testnet at some defined point. It then inherits the entire state and txpool from goerli, so you have actual state to test against. This is super useful for sync testing and allows us to simulate load properly. 

We'd eventually do this for mainnet as well, this would give us a lot of certainty that things behave as expected post merge, especially with non-finality and chaotic scenarios. 

I've added a bit of automation in creating such a fork genesis state here: https://github.com/skylenet/ethereum-genesis-generator/tree/goerli-shadow-fork. I'll probably test it next week :)

Currently we just want to see how things work, we'd eventually make it similar to kintsugi (with docs, test ideas, guides, etc) so the public can join in and help find bugs :)

Reply:
Ahh got it, very cool and interesting - major thanks for the insight Pari! Will definitely look over all of the above and duly noted re: the testing setup/situation :)